### ISSUE TITLE
surprising behavior: --verify-module doesn't verify submodules

### ISSUE THREAD

**Issue by jonhnet** [verus-lang/verus#596]
I've been using --verify-module to focus verus' attention on one file (module) at a time. That file contains state machines, which create submodules, and --verify-module ignores those submodules. So my verification has been surprisingly ignoring a big chunk of my file. That's certainly not what this user expected; I doubt it's what other users will expect.

**Comment by jaybosamiya** [comment]
Relevant: https://github.com/verus-lang/verus/discussions/385

Fwiw, I've hit upon the same issue in the past, thus verus-mode.el has an [explicit check](https://github.com/verus-lang/verus-mode.el/blob/main/verus-mode.el#L401-L404) for this situation where it will disable its "only run on the current module" behavior and starts running on the whole module.

I had tried implementing a "module and its submodules" wildcard but had hit some weird condition that caused me to abandon the approach I was taking; not sure what it was and I can't seem to find my notes on it; anyways relevant bit of Verus to update to support wildcard modules would be around [here](https://github.com/verus-lang/verus/blob/7507c4be8cb5f32f1ec2964da0c09766f7fa63d2/source/rust_verify/src/verifier.rs#L1161C1-L1200). I don't see anything that should prevent a "module and its submodules" implementation by just updating this to also handle a wildcard case, so am not sure why I had abandoned the approach I was taking back then. Could potentially have just been that I thought a nicer front-end interface might be possible or something, but yeah, I think this is feasible to implement with the wildcard thing, and I think (even if we don't yet have a better CLI option for it); at minimum we need something that tells us what is happening.
